src/pages/SignUp/OauthSignUp.style.ts

```diff
@@ -0,0 +1,63 @@
+import { css } from "@emotion/react";
+
+import { theme } from "@/styles/theme";
+
+export const OauthSignUpStyle = {
```

Comment:
OauthSignUpStyle는 일반 객체니 카멜 케이스를 따르는게 일관성 있어서 좋을 것 같습니다! 
```suggestion
export const oauthSignUpStyle = {
```